public/script.js

```diff
@@ -1 +1,49 @@
-console.log('tudo bem?');
\ No newline at end of file
+var btnAddItem = document.getElementById("button-add-item");
+var numberItem = 1;
+
+//functions
+function addItem() {
+	//Declarações:
+	var valueInput = document.getElementById("input-add-item").value;
```

Comment:
É aconselhável criar variáveis apenas quando for preciso retulizar o valor, mais de uma vez.
Nesse caso, como o valor só é usado em um momento, recomenda-se que seja atribuido diretamente, quando for utilizado:

```js
 label.innerHTML = document.getElementById("input-add-item").value;
```

A exeção é quando uma refatoração for necessária, para reduzir a complexidade do código ou o tamanho de uma linha, dividindo as etapas. Ou quando o valor só é acessível dentro de um escopo e precisa ser usado em outro. Nesse caso, o uso de uma variável que vai ser utilizada só uma vez, é válido.